packages/rum/test/viewCollection.spec.ts

```diff
@@ -191,6 +197,45 @@ describe('rum track url change', () => {
     window.location.hash = '#bar'
   })
 
+  it('should not create a new view when it is an Anchor navigation', (done) => {
+    const { lifeCycle } = setupBuilder.build()
+    mockGetElementById()
+    lifeCycle.subscribe(LifeCycleEventType.VIEW_CREATED, createSpy)
+
+    function hashchangeCallBack() {
+      expect(createSpy).not.toHaveBeenCalled()
+      window.removeEventListener('hashchange', hashchangeCallBack)
+      done()
+    }
+
+    window.addEventListener('hashchange', hashchangeCallBack)
+
+    window.location.hash = '#testHashValue'
+  })
+
+  it('should acknowledge the view location hash change after an Anchor navigation', (done) => {
+    const { lifeCycle } = setupBuilder.build()
+    const spyObj = mockGetElementById()
```

Comment:
what about something more expressive than `spyObj`?

Reply:
as we just "clear" it, I did looked for a better name. Also the comment helps understanding the usage of `spyObj.and.callThrough()`

Reply:
it could have been an opportunity to be more expressive